ui/inspectors/directory-services.reel/winbind-service.reel/winbind-service.html

```diff
@@ -88,6 +88,23 @@
             }
         },
 
+        "saslWrapping": {
+            "prototype": "blue-shark/ui/field-select.reel",
+            "properties": {
+                "element": {"#": "saslWrapping"},
+                "label": "SASL Wrapping",
+                "options": [
```

Comment:
you can refer to this as an example: https://github.com/freenas/gui/blob/master/ui/sections/system/inspectors/alert.reel/table-predicates.reel/table-row-predicate.reel/table-row-predicate.js#L29